frontend/src/app/jwt-interceptor.service.ts

```diff
@@ -0,0 +1,17 @@
+import { Injectable } from '@angular/core';
+import { HttpInterceptor, HttpRequest, HttpEvent, HttpHandler } from '@angular/common/http';
+import { Observable } from 'rxjs/Observable';
+
+@Injectable()
+export class JwtInterceptorService implements HttpInterceptor {
+
+  token: string;
```

Comment:
True. But since I think we would have to change a whole lot of other things, I didn't take that into account.